src/collection/components/CollectionFragment.tsx

```diff
@@ -294,6 +294,8 @@ const CollectionFragment: FunctionComponent<CollectionFragmentProps> = ({
 										onInit={initFlowPlayer}
 										start={cuePoints.start}
 										end={cuePoints.end}
+										publishedAt={'30-12-2011'}
+										publisher={'VRT'}
```

Comment:
dit is wel vreemd dat je publishedAt en publisher moet meegeven aan een video.
Kan je dit niet algemener benoemen zoals: title
of title links, titel rechts

of worst case moeten we werken met een "slot"

publishedAt en publisher maakt de flowplayer component niet erg herbruikbaar buiten onze app.